packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -55,13 +56,16 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
         private readonly tokenProvider: ITokenProvider,
         driverPolicies: Partial<IRouterliciousDriverPolicies> = {},
     ) {
+        // 5 days is the max allowed value per the IDocumentStorageServicePolicies.maximumCacheDurationMs policy
+        const snapshotCacheExpiryMs: FiveDaysMs = 432000000;
```

Comment:
I wonder if we can move this to some constant file that's shared across all drivers without creating a breaking change.

Reply:
Eh, I could export it from somewhere in driver-utils, but I don't think it's worth it.  The type is in driver-definitions, and if you type any number besides this it won't compile.